src/app/settings/organization-plans.component.html

```diff
@@ -209,8 +210,16 @@ <h2 class="spaced-header mb-4">{{'paymentInformation' | i18n}}</h2>
         </ng-container>
         <small class="text-muted font-italic mt-2 d-block" *ngIf="!createOrganization">
             {{'paymentCharged' | i18n : (interval | i18n) }}</small>
-    </ng-container>
-    <div [ngClass]="{'mt-4': !createOrganization || plans[plan].noPayment}">
+    </div>
+    <div class="mt-4">
```

Comment:
I am wondering if we should toggle this off until an API is chosen and implemented? Right now it just shows on screen that sales tax is 0$.